README.md

```diff
@@ -1,13 +1,16 @@
 ## i-MSCP - internet Multi Server Control Panel
 
-i-MSCP - internet Multi Server Control Panel is an open-source software which allows to manage shared hosting
-environments on Linux servers.
+i-MSCP - internet Multi Server Control Panel is an open-source software which allows to manage shared hosting environments on Linux servers.
```

Comment:
I'd say either

"i-MSCP - internet Multi Server Control Panel is open-source software which allows to manage shared hosting environments on Linux servers."

or

"i-MSCP - internet Multi Server Control Panel is an open-source software package which allows to manage shared hosting environments on Linux servers."